.env

```diff
@@ -0,0 +1,3 @@
+GLOBAL_CLIENT_ID=pSm3810KgGx5qnjckjRGoWuBLlU0Hjgl
+GLOBAL_CLIENT_SECRET=OG1G34icGnYQKjtV26Q_qokQe6bafT8K3ecdHwIQ4pcCEiKjfQNBpZkzJdVxg_8s
+DOMAIN=happyr.eu.auth0.com
```

Comment:
try to avoid pushing the credentials. add this to the `.gitignore` and rotate your secret please